CrossPlatformDesktopProject/CrossPlatformDesktopProject/NPC/Bat/BatWalkSouth.cs

```diff
@@ -36,14 +36,18 @@ public void Draw(SpriteBatch spriteBatch, float xPos, float yPos)
 
         public void Update()
         {
+            if (bat.xPos == 450 && bat.yPos == 150)
```

Comment:
These statments seem awfully hard-coded; maybe keeping track of how long to go rather than where to stop would be better. But we can fix that later.

Reply:
Yeah they're pretty ugly and I was trying to get it to work without hard code but it wasn't going too well, I'll continue to look at it today though.